.git-commit-template

```diff
@@ -5,5 +5,38 @@ Explanation
 Resolves:
 https://github.com/SSSD/sssd/issues/XXXX
 
+# If a release note is required, choose one of the tags and place it here.
+# See the description bellow for more tags and information.
+:relnote: Generic release note.
+:feature: New feature desription.
+:bug: Notable bug desription.
```

Comment:
How about add something like (in some shorter version):
:refactoring:
:optimization:
:documentation"
:backport:
:0day:

For example how to classify now :deloption: ? This is not a :bug: nor :feature: but good candidate for :refactor:

Reply:
Hmmm, interesting, I am wondering if there will be ": bug :" tag in commit message it will be evaluated into emoji icon too.

Reply:
refactoring -> generic release note -> :relnote:
optimization -> generic release note -> :relnote:
documentation, this really doesn't require release note. If it is such a huge change then :relnote:
backport doesn't make sense (backport will also bring the tags in the commit message so it already contains it)
0day is not upstream stuff

The option tags should be used on its own. The intention is not to use the tags without a message but always with a message. E.g.:

```
NOT
:relnote: New feature description
:feature:

YES
:feature: New feature description.
```

The option tags are little bit specific to make sure the message is always consistent, e.g.:

```
:newoption: ldap_cool_new_option, sssd-ldap

Release notes:
* Added option `ldap_cool_new_option`, see `man sssd-ldap` for description.
```

Not all commits need to produce release notes (actually the majority of them should not). See [latest release notes](https://sssd.io/docs/users/relnotes/notes_2_3_1.html) for example.

Reply:
Only `:bug:` is affected from those words so we can use something else. :bugfix:?

Reply:
I'd prefer to use :bugdesc: as the field is bug description.

By the way, in .git-commit-template there are several places where desription is written instead of des**C**ription.

Reply:
How about use :tag: but change tag keyword to include prefix. This will break this emoji related interpretation?
:sss_bug:
:sss_feature:
:sss_relnote: